bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -1881,6 +1881,12 @@ func (gcp *Gcp) ConfigPodDefault() error {
 		}
 	}
 
+	getReq := crdClient.Get().Resource(mapping.Resource.Resource).Namespace(defaultNamespace).Name("add-gcp-secret")
```

Comment:
Nit define a constant for the name and use it here and here
https://github.com/kubeflow/kubeflow/blob/0a370635ad1db8139a1cf56bef329f34344fea6d/bootstrap/pkg/kfapp/gcp/gcp.go#L1761

Reply:
done